main.go

```diff
@@ -68,6 +69,7 @@ func main() {
 	tokenExpiration := flag.Int64("token-expiration", pkg.DefaultTokenExpiration, "The token expiration")
 	region := flag.String("aws-default-region", "", "If set, AWS_DEFAULT_REGION and AWS_REGION will be set to this value in mutated containers")
 	regionalSTS := flag.Bool("sts-regional-endpoint", false, "Whether to inject the AWS_STS_REGIONAL_ENDPOINTS=regional env var in mutated pods. Defaults to `false`.")
+	watchConfigMap := flag.Bool("watch-config-map", false, "Whether to watch the pod-identity-webhook ConfigMap for additional Service Accounts. Defaults to `false`.")
```

Comment:
Could mention that it's in namespace `--namespace` and description for `--namespace` should also  be updated since it's being overloaded/reused now

Reply:
Maybe also add "these service accounts will be watched even if they do not have annotations"

Reply:
I tried to add something here. Kept it brief so that the usage docs for the relevant flags do not get too long.